src/style-flight-main.scss

```diff
@@ -5,9 +5,11 @@
 
 // Import Auro tokens
 @import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import '../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
```

Comment:
@aesthetiques is there a reason why the path to the node_modules is different here?

Reply:
My question is... why was this and line 12 added? I am not seeing any changes in the Sass that require these references.

Reply:
Same as other